aws/elasticsearch/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "domain_name" {
+  default = "es-domain"
+}
+
+variable "es_version" {
+  default = "7.1"
+}
+
+variable "es_instance_type" {
+  default = "t2.small.elasticsearch"
+}
+
+variable "es_instance_count" {
+  default = "1"
+}
+
+variable "ebs_volume_size" {
+  default = "50"
+}
+
+variable "subnet_ids" {
```

Comment:
You should set the type to a list and remove the `[]` from `main.tf` instead

Reply:
done